server/openapi.go

```diff
@@ -234,7 +235,7 @@ func (s *Server) schemaDiff(c echo.Context) error {
 		return echo.NewHTTPError(http.StatusInternalServerError, "Failed to parse target schema into AST nodes").SetInternal(err)
 	}
 
-	diff, err := parser.SchemaDiff(sourceSchemaNodes, targetSchemaNodes)
+	diff, err := pg.SchemaDiff(sourceSchemaNodes, targetSchemaNodes)
```

Comment:
This diff actually breaks semantics here, previously the `parser.SchemaDiff` is database-agonistic but not anymore. Though not a problem since we only claim we support PostgreSQL for now.